daffodil-runtime1/src/main/scala/org/apache/daffodil/infoset/XMLTextInfosetOutputter.scala

```diff
@@ -25,18 +25,23 @@ import org.apache.daffodil.dpath.NodeInfo
 import org.apache.daffodil.exceptions.Assert
 import org.apache.daffodil.util.Indentable
 
+import org.apache.daffodil.api.XMLOutputStyle
+
+class NoMatchException(s: String) extends Exception(s) {}
```

Comment:
I'm not sure we need the null check, bu if we do, I'd suggest using an existing exception. I think `Assert.usage` might be similar to what we already use? E.g.
```scala
Assert.usage(xmlOutputStyle != null, "message here")
```